recipe/meta.yaml

```diff
@@ -141,14 +136,14 @@ outputs:
         - pip                             # Required for in tree builds
         - pkg-config 0.29.2               # [unix]
         - python
-        - pyyaml 6.0
-        - requests 2.28.1
+        - pyyaml 6.0.1
+        - requests 2.31.0
         # CF: PyTorch relies on features that were removed in later versions.
         - setuptools
         - sleef 3.5.1                     # [osx and arm64]
-        - typing-extensions 4.4.0
+        - typing_extensions 4.7.1
```

Comment:
How about `typing_extensions 4.9.0` or even `typing_extensions`? In runtime, there is already typing_extensions >=4.8.0

Reply:
oh right yeah, it's a pure python package. sure I'll remove the pinning. not sure why it's there tbh. Upstream wants >=4.8.0 hence the run pinning.